tensorflow/c/tf_attrtype.h

```diff
@@ -24,12 +24,13 @@ typedef enum TF_AttrType {
   TF_ATTR_STRING = 0,
   TF_ATTR_INT = 1,
   TF_ATTR_FLOAT = 2,
-  TF_ATTR_BOOL = 3,
-  TF_ATTR_TYPE = 4,
-  TF_ATTR_SHAPE = 5,
-  TF_ATTR_TENSOR = 6,
-  TF_ATTR_PLACEHOLDER = 7,
-  TF_ATTR_FUNC = 8,
+  TF_ATTR_DOUBLE = 3,
```

Comment:
I believe this has to be ABI stable, let's not change the existing values and add `TF_ATTR_DOUBLE` at the end of this list.